src/workerd/util/sqlite.h

```diff
@@ -413,6 +414,7 @@ class SqliteDatabase::Query final: private ResetListener {
         regulator(regulator),
         ownStatement(db.prepareSql(regulator, sqlCode, 0, MULTI)),
         maybeStatement(ownStatement) {
+    resetRowCounters();
```

Comment:
This line isn't needed because in this constructor the statement is always newly-constructed so the counts will already be zero.

Only the other constructor, which takes a `Statement&` that may have been used before, needs to reset the counters.